### DISCUSSION TITLE
Recover notes from old installation? (no backup)

### DISCUSSION THREAD

**Discussion by NorPhi** [TriliumNext/Trilium#2083] [General]
My PC got taken out by a power surge. This was just before the next scheduled backup and I've digitalised a lot of handwritten notes since the last backup. The SSD my /home directory was on survived. Is it save to just copy the old ~/.local/share/trilium-data over that of a new installation or how should I proceed?

**Comment by zadam** [comment]
Yes, all data is stored in the `trilium-data` directory so restoring that should be sufficient.
